### ISSUE TITLE
Add text to clarify that profile URIs don't have to resolve

### ISSUE THREAD

**Issue by larsgsvensson** [w3c/dxwg#1000]
Regarding the point, that profile URIs don't necessarily have to resolve, @tombaker noted: 

> That's an interesting point. The URIs in RDF triples also do not have to resolve. Moreover, a URI in an RDF triple that resolves today may not resolve five years from now (owner moved on, DNS fee not paid, project finished...), but that does not make the triple invalid.
The same thing could happen to a profile URI, but it could continue to serve its purpose for CONNEG, so perhaps it is indeed worth pointing this out.

We should add some text to clarify this.

**Comment by andrea-perego** [comment]
@larsgsvensson , @nicholascar , @rob-metalinkage ,

Should this issue be transferred to https://github.com/w3c/dx-connegp ?

**Comment by larsgsvensson** [comment]
@andrea-perego 
> Should this issue be transferred to https://github.com/w3c/dx-connegp ?

Yes, I think it should. I've seen that you have transferred other issues to connegp, too, so can you please transfer this one as well?

Thanks,
Lars
